### ISSUE TITLE
Compute statistics about samples

### ISSUE THREAD

**Issue by lmove** [akago/breaking_updates_rl#2]
## Description
Currently we are generating the CSV files to sample the breaking dependency upgrades but we are not storing any information about the distribution of the labels (i.e., results of the $chi^2$ test) between the training and test sets for each round, nor information about missed rounds. I recommend including this information for self containment purposes.

## Solution
In the **Test/Train Split** section of the notebook, create a new file that stores metadata about the runs to compute the samples. This file should include:
- Test set size
- Training set size
- Test set file path
- Training set file path
- $chi^2$ coefficient
- p-value
- Cramer's V (effect size)
- Attempted rounds
- Failing rounds (should be attempted - 1)
